raiden/ui/cli.py

```diff
@@ -746,6 +757,26 @@ def run(ctx, **kwargs):
     from raiden.ui.console import Console
     from raiden.api.python import RaidenAPI
 
+    if kwargs['config_file']:
+        try:
+            config_file = toml.load(kwargs['config_file'])
+            addr_options = {'address', 'registry_contract_address', 'discovery_contract_address'}
+            for option_name, option_value in config_file.items():
+                if option_name in addr_options:
+                    option_value = address_checksum_and_decode(option_value)
+
+                if (option_name not in DEFAULT_OPTION or
+                    (kwargs[option_name] and
+                        kwargs[option_name] != DEFAULT_OPTION[option_name])):
+                    print('Use CLI option {} value {}'.format(option_name, kwargs[option_name]))
```

Comment:
`print()` is a bit too nosy here. Please use `log.debug()`.

Reply:
since this print is before configuring logging, i removed it